contrib/misc/bareos_pam_integration/README.rst

```diff
@@ -85,7 +85,9 @@ Configure the Bareos Director as described by https://docs.bareos.org/master/Tas
 
 Create a bconsole configuration file, name it :file:`bconsole-pam.conf`.
 
-Test to connect via bconsole to the bareos-dir::
+Test to connect via bconsole to the bareos-dir.
+
+::
```

Comment:
Why have your changed this? I thought, the original formatting is the preferred from of RST, see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html
Especially, the column after `Test to connect via bconsole to the bareos-dir` is intentionally.

Reply:
It depends what you are going for, for source readability I personally would prefer the expanded form of literal blocks. But I'm going to revert to the fully minimized.

Reply:
Reverted.